Description: Comment out portions using the gem "turn"
 Removing the dependecy on turn as it is not and can't be packaged.
 .
 ruby-fog-xml (0.1.1-1) UNRELEASED; urgency=medium
 .
   * Initial release (Closes: #nnnn)
Author: Balasankar C <balasankarc@autistici.org>


--- ruby-fog-xml-0.1.1.orig/spec/minitest_helper.rb
+++ ruby-fog-xml-0.1.1/spec/minitest_helper.rb
@@ -1,23 +1,23 @@
 require "minitest/spec"
 require "minitest/autorun"
-require "turn"
+#require "turn"
 require "excon"
 require "fog/core"
 
-Turn.config do |c|
-  # use one of output formats:
-  # :outline  - turn's original case/test outline mode [default]
-  # :progress - indicates progress with progress bar
-  # :dotted   - test/unit's traditional dot-progress mode
-  # :pretty   - new pretty reporter
-  # :marshal  - dump output as YAML (normal run mode only)
-  # :cue      - interactive testing
-  # c.format  = :outline
-  # turn on invoke/execute tracing, enable full backtrace
-  c.trace   = 20
-  # use humanized test names (works only with :outline format)
-  c.natural = true
-end
+#Turn.config do |c|
+  ## use one of output formats:
+  ## :outline  - turn's original case/test outline mode [default]
+  ## :progress - indicates progress with progress bar
+  ## :dotted   - test/unit's traditional dot-progress mode
+  ## :pretty   - new pretty reporter
+  ## :marshal  - dump output as YAML (normal run mode only)
+  ## :cue      - interactive testing
+  ## c.format  = :outline
+  ## turn on invoke/execute tracing, enable full backtrace
+  #c.trace   = 20
+  ## use humanized test names (works only with :outline format)
+  #c.natural = true
+#end
 
 if ENV["COVERAGE"]
   require "coveralls"
